ENIGMAsystem/SHELL/Universal_System/var4.h

```diff
@@ -90,9 +91,11 @@ struct variant
   operator string() const;
   
   variant();
+  variant(void *p);
```

Comment:
This should technically be `const`, since we're going to store it const. Otherwise, we eliminate the option of passing a non-const pointer even though that's all we need.

Reply:
Fixed in commit 0c602f67dff817b7e596fa318e09048ce2696bec